docs/index.html

```diff
@@ -168,7 +169,7 @@ <h1 id="iban-info-title" class="text-2xl font-bold text-primary mb-4">IBAN Infor
         };
 
         function setLanguage(lang) {
-            const t = translations[lang] || translations.en;
+            const t = TRANSLATIONS[lang] || TRANSLATIONS[DEFAULT_LANGUAGE];
```

Comment:
Just asking for now, so don't commit anything yet: is it possible to remove the duplication of the fallbacking logic (`|| TRANSLATIONS[DEFAULT_LANGUAGE]`) too?